README.md

```diff
@@ -468,6 +469,72 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+```js
+  class MyClassName extends ANSWERS.Component {
+    constructor(config = {}, systemConfig = {}) {
+      super(config, systemConfig);
+
+      // Add your Handlebars template here.
+      this.setTemplate(`
+        <div class="myDirectAnswer">
```

Comment:
I think we should separate out this HTML into another code block. People can provide a template in-line like this, but they can also provide it in other ways. I think the best practice is to not have it inline like this. I think we want a bit more complexity in this example as well. Maybe show a basic switch on the type to do some very simple formatting of the answer value. One of the switch cases could further branch by checking for existence of a particular fieldName and altering formatting accordingly.

Reply:
What is best practice for how to provide a custom template? I'm not aware of any other ways to do this. This is how it's currently being done in the theme.

Reply:
I feel a little uncomfortable just glossing over how to add a template especially since the "Creating Custom Components" section of the readme also kind of glosses over this. I guess the other way would be to add your template directly to the handlebars renderer? Think we should add documentation on the best way to add a template.

Reply:
I wrote a small blurb that you can include your own template bundle and reference the template by name. Does this work?